packages/components/src/select-control/index.tsx

```diff
@@ -89,15 +115,17 @@ function SelectControl(
 				label={ label }
 				size={ size }
 				suffix={
-					<DownArrowWrapper>
-						<Icon icon={ chevronDown } size={ 18 } />
-					</DownArrowWrapper>
+					props.suffix || (
```

Comment:
Is passing `props.suffix` here  (and omitting it from the props passed to <Select />) a bug fix? If yes, should it be included in the changelog?

Reply:
It's not really a bug fix... there was no bug being caused earlier, but it was causing it impossible to truly document the full types of the component because we were spreading `props` into two places. By being explicit we can prove to TypeScript that the only props we spread into `InputBase` are truly the ones that it should have.

Reply:
Got it, thank you!